table-paste-col/table-paste-col.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+import sys
+import argparse 
+
+"""
+Add a column containing a constant value to a table. Header assumed. 
+"""
+def get_input_file_object(filename):
+    if filename == "-":
+        return sys.stdin
+    return open(filename, "r")
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument("-t", "--table", default='-', help="input csv file; STDIN if - (default: %(default)s")
+parser.add_argument("-c", "--colname", help="name for new column")
```

Comment:
better use --col-name to maintain the same style as the perl script.

make this required `, required=True`

Reply:
If I use "col-name" and refer to it later with args.col-value I get a namespace error

Reply:
you need args.col_value. argparse converts - to _